testgrid/config/config.yaml

```diff
@@ -3160,6 +3164,8 @@ dashboards:
     test_group_name: ci-kubernetes-e2e-kubeadm-gce-1-7-on-1-8
   - name: kubeadm-gce-upgrade-1.7-1.8
     test_group_name: periodic-kubernetes-e2e-kubeadm-gce-upgrade-1-7-1-8
+  - name: periodic-ci-kubernetes-e2e-kubeadm-pinned-releases
```

Comment:
We might wanna put this here as well? https://k8s-testgrid.appspot.com/sig-release-master-blocking